samples/subsys/mgmt/hawkbit/prj.conf

```diff
@@ -14,6 +14,7 @@ CONFIG_HTTP_CLIENT=y
 CONFIG_DNS_RESOLVER=y
 CONFIG_JSON_LIBRARY=y
 CONFIG_BOOTLOADER_MCUBOOT=y
+CONFIG_SETTINGS=y
```

Comment:
we probably don't need the `CONFIG_NVS=y` above anymore?

Reply:
We need a backend for the settings subsystem enabled. 
Therefore we use `CONFIG_NVS=y `. 
See https://docs.zephyrproject.org/latest/kconfig.html#CONFIG_SETTINGS_BACKEND